lib/serializers/resource-serializer.js

```diff
@@ -256,7 +256,13 @@ class ResourceSerializer extends EsSerializer {
     if (this.object._items) {
       // Add `[bibid]-[itemid]` to bib-level `uris` property for each item to aid retrieval of unified bib-item object
       this.statements.uris = this.statements.uris.concat(this.object._items.map((i) => [this.statements.uri, i.id].join('-')))
-
+      // Extract number of electronic resources. These are previously combined into one item with electronic locator links in the discovery store poster
+      this.object._items.forEach((item) => {
+        if (item.has('nypl:electronicLocator') || item.has('bf:electronicLocator')) {
```

Comment:
I see now I had this wrong: The namespace should be `bf:`, not `nypl:`. So you should be able to use `bf:electronicLocator` for both the `.has` and `.statements` call.